### PR TITLE
Add a bio for JSC

### PR DESCRIPTION
**Pull request by JoshuaSCochrane**
Do stuff!!! Self descriptive. _md?_

To accept this request, press the <kbd>Any</kbd> key.

### CODE REVIEW

File: docs/JoshuaSCochrane.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Josh Cochrane
+
+Hometown: Watertown, WI
```
Comment (HarryPalz) [inline docs/JoshuaSCochrane.md:3]: Sorry to hear this.

File: docs/JoshuaSCochrane.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+Occupation: Firmware Engineer
```
Comment (kphutt) [inline docs/JoshuaSCochrane.md:5]: This is all wrong, start over from scratch.

File: docs/JoshuaSCochrane.md (hunk 1 above)
Comment (ch-rob) [inline docs/JoshuaSCochrane.md:3]: @JoshuaSCochrane  Could you be more specific on your hometown... maybe your addy, blood type, first born, mother's maiden name, last for of your SSN, etc, etc

**Comment by HarryPalz** [comment]
test

**Comment by JoshuaSCochrane** [comment]
:-1: :-1:

**Comment by JoshuaSCochrane** [comment]
@HarryPalz I was going to revert your sniped merge/close, but meh; I'll allow it. This time.

**Comment by HarryPalz** [comment]
@JoshuaSCochrane got you good

**Comment by JoshuaSCochrane** [comment]
@HarryPalz forgot to mention that this fixes #24

**Comment by JoshuaSCochrane** [comment]
Or fixes #24 ?

**Comment by HarryPalz** [comment]
@JoshuaSCochrane you can also close issues in the issue list! :relieved:

**Comment by JoshuaSCochrane** [comment]
@HarryPalz Yeah, that's what I ended up doing; wasn't sure why it wasn't picking it up, but w/e

**Comment by HarryPalz** [comment]
@JoshuaSCochrane According to @pdgessler , it needs to be in the close comment, not just any ole comment

**Comment by pdgessler** [comment]
@JoshuaSCochrane @HarryPalz the issue is the issue mention has to be issued in a commit message, not a comment, in order to issue the request to close the issue.

**Pull request merged**